### PR TITLE
ALBS-449: Implement Admin pages

### PR DESCRIPTION
**Pull request by javihernandez**
Depends on https://github.com/AlmaLinux/albs-web-server/pull/289

### CODE REVIEW

**Review by maccelf — APPROVED** [review]
LGTM

**Review by soksanichenko — APPROVED** [review]

File: src/components/UserRolesEditor.vue [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+                v-model="teamModel"
+                :options="options"
+                label="Select team"
+                use-input
+                input-debounce="0"
+                @update:model-value="onTeamSelected"
+                @filter="teamSelectFilter"
+            />
+
+            <SlideTransition>
```
Comment (maccelf) [inline src/components/UserRolesEditor.vue:23]: Why you don't use `q-slide-transition` https://quasar.dev/vue-components/slide-transition ?

**Review by anfimovdm — APPROVED** [review]

File: src/components/UserRolesEditor.vue (hunk 1 above)
Comment (javihernandez) [reply src/components/UserRolesEditor.vue:23]: I totally agree with you, but for some reason that I don't understand, it doesn't work. Maybe is related to the fact that the table is inside a dialog, or that maybe I'm playing around with `height/max-height` properties, can't tell, sorry.
If you have any advice or idea on how to make it work I would appreciate. :man_shrugging:

Comment (maccelf) [reply src/components/UserRolesEditor.vue:23]: No, i don't have any idea 🥲  
So it works now, but i think in the future we should figure it out

**Review by maccelf — APPROVED** [review]
👍🏼

**Review by anfimovdm — APPROVED** [review]

File: src/components/UserRolesEditor.vue (hunk 1 above)
Comment (javihernandez) [reply src/components/UserRolesEditor.vue:24]: Totally agree with you. For the record, I tested different combinations and also, tried using a different version of Quasar, none of it made it work :disappointed: 
In any case, I left a TODO note for our future selves :smile:

**Pull request merged**